### PR TITLE
Rating and snackbar 

### PR DESCRIPTION
**Pull request by Bibin-Antony**

### CODE REVIEW

**Review by hemanshthakur — APPROVED** [review]
looks good to  go

**Comment by Bibin-Antony** [comment]
image

image

**Review by ranitasahaa1230 — CHANGES_REQUESTED** [review]
Anything less than 10px use px, not a bug, its just a good practice

File: Container/HTML/rating.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="/Container/CSS/rating.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (ranitasahaa1230) [inline Container/HTML/rating.html:8]: You can add description in the meta tag-author description, about the page content

**Review by ranitasahaa1230 — COMMENTED** [review]
You can add a description in the meta tag-author description, about the page content

**Pull request merged**